types.go

```diff
@@ -59,6 +59,8 @@ type Config struct {
 	//
 	// Theoretically this would go away if/when we standardize on OCI.
 	UnpackRootfs bool `json:"unpack_rootfs" envconfig:"optional"`
+
+	AddHosts string `json:"add_hosts" envconfig:"ADD_HOSTS,optional"`
```

Comment:
Since this is a Buildkit-specific feature, can this be called `BUILDKIT_ADD_HOSTS`?